packages/FormElement/stories/FormElement.stories.mdx

```diff
@@ -0,0 +1,265 @@
+  <AccessibilityExample />
+</FormElementStory>
+
+### Alternate Layouts
+
+The FormElement can be used in a variety of formats. These can include FormElements with checkboxes, radios, listbox's etc.
+
+#### Checkboxes
+
+```jsx
+<FormElement hasFieldSet label="Form Label">
+  <FormElement.Content>
+    {({ ariaDescribedBy: ariaDescribedByOuter }) => getCheckboxOptions(ariaDescribedByOuter)}
+  </FormElement.Content>
+  <FormElement.Description>
+    <span>Description of this field.</span>
+  </FormElement.Description>
+  <FormElement.Error>{errorText}</FormElement.Error>
+  <FormElement.Help>
+    Give me some help. <a href="wegalvanize.com">Learn more</a>.
+  </FormElement.Help>
+</FormElement>
+```
+
+##### Result
+
+<CheckboxExample />
+
+#### Radios
+
+```jsx
+<FormElement hasFieldSet label="Form Label">
+  <FormElement.Content>
+    {({ ariaDescribedBy: ariaDescribedByOuter }) => getRadioOptions(ariaDescribedByOuter)}
+  </FormElement.Content>
+  <FormElement.Description>
+    <span>Description of this field.</span>
+  </FormElement.Description>
+  <FormElement.Error>{errorText}</FormElement.Error>
+  <FormElement.Help>
+    Give me some help. <a href="wegalvanize.com">Learn more</a>.
+  </FormElement.Help>
+</FormElement>
+```
+
+##### Result
+
+<RadioExample />
+
+#### Date Picker
+
+```jsx
+<FormElement label="Form Label">
+  <FormElement.Content>
+    {({ idForLabel, ariaDescribedBy }) => (
+      <DatePicker onError={() => {}} hasError={Boolean(errorText.length)} id={idForLabel} onChange={() => {}}>
+        <DatePicker.Input aria-describedby={ariaDescribedBy} />
+      </DatePicker>
+    )}
+  </FormElement.Content>
+  <FormElement.Description>
+    <span>Description of this field.</span>
+  </FormElement.Description>
+  <FormElement.Error>{errorText}</FormElement.Error>
+  <FormElement.Help>
+    Give me some help. <a href="wegalvanize.com">Learn more</a>.
+  </FormElement.Help>
+</FormElement>
+```
+
+##### Result
+
+<DatePickerExample />
+
+#### ListBox
+
+```jsx
+<FormElement label="Form Label">
+  <FormElement.Content>{({ refLabel }) => <ListBox refLabel={refLabel}>{listboxOptions}</ListBox>}</FormElement.Content>
+  <FormElement.Description>
+    <span>Description of this field.</span>
+  </FormElement.Description>
+  <FormElement.Help>
+    Give me some help. <a href="wegalvanize.com">Learn more</a>.
+  </FormElement.Help>
+</FormElement>
+```
+
+##### Result
+
+<ListBoxExample />
+
+#### ButtonGroup
+
+```jsx
+<FormElement label="Form Label" onClickLabel={() => refButtonGroup.current.focus()}>
+  <FormElement.Content>
+    {() => <ButtonGroup ref={refButtonGroup}>{buttonGroupOptions}</ButtonGroup>}
+  </FormElement.Content>
+  <FormElement.Description>
+    <span>Description of this field.</span>
+  </FormElement.Description>
+  <FormElement.Help>
+    Give me some help. <a href="wegalvanize.com">Learn more</a>.
+  </FormElement.Help>
+</FormElement>
+```
+
+##### Result
+
+<ButtonGroupExample />
+
+#### Nested FormElement
```

Comment:
This example has a lot of code, let's just explain that it's possible to nest the FormElement in another FormElement, why you might want to do that, and then link to the NestedExample story or github source.

Reply:
While I don't think this Nested example is a common enough use case to include in the Docs, I do think that the `ComponentLabel` _is_ a useful example to include here.  Can you include it (with a brief 1-2 sentence explanation before the code block example?)